runtime/Go/antlr/tree.go

```diff
@@ -28,7 +28,7 @@ type SyntaxTree interface {
 type ParseTree interface {
 	SyntaxTree
 
-	Accept(Visitor ParseTreeVisitor) interface{}
+	Accept(visitor ParseTreeVisitor)
```

Comment:
Accept should only get called by the generate code, not by visitor implementations.